Add token to browser

xendk opened the pull request:
Add token at login.. Untested as we're still waiting on a proper clientId.

DDBLM-25

xendk committed:
Add openplatform token to browser at login

kasperg reviewed (changes requested):
:+1: but note the build error:

The installation fails with the following error: `DingProviderNoProvider: Provider module not configured for provider type openplatform_token. in ding_provider_function() (line 582 of /home/circleci/drupal/profiles/ding2/modules/ding_provider/ding_provider.module).`

kasperg committed:
Make Openplatform token provider non-required

If we require this module we in essence also require 
ding_adgangsplatformen which we do not want to do at this point in
time.

Consequently we make the provider non-required and handle
situations where the provider is not available as elegantly as
possible.

xendk reviewed (commented):
:+1:
